client/src/components/main/shared/GeneAutocomplete/GeneAutocomplete.tsx

```diff
@@ -2,84 +2,184 @@ import React, { useState, useEffect } from "react";
 import { TextField } from "@material-ui/core";
 import Autocomplete from "@material-ui/lab/Autocomplete";
 import { getGeneId, getGeneSuggestions } from "../../../../services/main";
+import { CSSProperties } from "@material-ui/core/styles/withStyles";
+import {
+  NormalizeGeneResponse,
+  SuggestGeneResponse,
+} from "../../../../services/ResponseModels";
+
+export enum GeneSuggestionType {
+  alias = "Alias",
+  symbol = "Symbol",
+  prevSymbol = "Previous Symbol",
+  none = "",
+}
+export type SuggestedGeneOption = { value: string; type: GeneSuggestionType };
+
+const defaultGeneOption: SuggestedGeneOption = {
+  value: "",
+  type: GeneSuggestionType.none,
+};
 
 interface Props {
-  selectedGene: string;
-  setSelectedGene: CallableFunction;
+  gene: string;
+  setGene: CallableFunction;
   geneText: string;
   setGeneText: CallableFunction;
-  style: unknown;
+  style: CSSProperties;
 }
 
 export const GeneAutocomplete: React.FC<Props> = ({
-  selectedGene,
-  setSelectedGene,
+  gene,
+  setGene,
   geneText,
   setGeneText,
   style,
 }) => {
-  const [geneOptions, setGeneOptions] = useState([]);
+  const [geneOptions, setGeneOptions] = useState<SuggestedGeneOption[]>([]);
+  const [geneValue, setGeneValue] = useState(
+    gene ? { value: gene, type: GeneSuggestionType.symbol } : defaultGeneOption
```

Comment:
nitpick- you could create a new variable called defaultGeneValue:
const defaultGeneValue = gene ? { value: gene, type: GeneSuggestionType.symbol } : defaultGeneOption

and then do useState(defaultGeneValue) for this and the next line!